packages/pods-core/src/v2/pods/export/AirtableExportPodV2.ts

```diff
@@ -59,11 +65,28 @@ class AirtableUtilsV2 {
         try {
           const _records = await func(record);
           return _records;
-        } catch (error) {
+        } catch (error: any) {
+          let message;
+          if (error?.statusCode === 422) {
```

Comment:
Instead of magic number 422, can we use Status Codes lib? 

`import { StatusCodes } from "http-status-codes";`